desktop/apps/consignments2/stylesheets/landing/desktop.styl

```diff
@@ -173,16 +212,25 @@
 
 .consignments2-upcoming-sales
   height 480px
-  padding-top 60px
   color white
+  padding-top 60px
+  background-size cover
+  background-position right
+
+  &--mobile
+    @media screen and (min-width: 641px)
```

Comment:
... and if not a mixin, some kind of constant that defines widths so that things are consistent across apps?